main.go

```diff
@@ -6,9 +6,38 @@ import (
 	"io/ioutil"
 	"log"
 	"net/http"
+	"database/sql"
+	_ "github.com/lib/pq"
 )
 
-func main() {
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = "postgres"
+	password = "1234"
+	dbname   = "test"
+)
+
+var(
+    db *sql.DB
+)
+
+func main() {	
+	var err error
+	psqlInfo := fmt.Sprintf("host=%s port=%d user=%s "+
```

Comment:
Would be better here to use `fmt.Sprintf` instead of using `+` for string interpolation, because it's faster